packages/oidc-client/src/lib/client.store.ts

```diff
@@ -1,42 +1,61 @@
+/*
+ * Copyright (c) 2025 Ping Identity Corporation. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
 import { CustomLogger, logger as loggerFn, LogLevel } from '@forgerock/sdk-logger';
 import { createAuthorizeUrl } from '@forgerock/sdk-oidc';
+import { createStorage, StorageConfig } from '@forgerock/storage';
+import { Micro } from 'effect';
+import { exitIsSuccess } from 'effect/Micro';
 
+import { authorizeµ } from './authorize.request.js';
 import { createClientStore } from './client.store.utils.js';
+import { createValuesµ, handleTokenResponseµ, validateValuesµ } from './exchange.utils.js';
+import { GenericError } from './error.types.js';
+import { oidcApi } from './oidc.api.js';
 import { wellknownApi, wellknownSelector } from './wellknown.api.js';
 
 import type { ActionTypes, RequestMiddleware } from '@forgerock/sdk-request-middleware';
 import type { GetAuthorizationUrlOptions } from '@forgerock/sdk-types';
 
 import type { OidcConfig } from './config.types.js';
-import { GenericError } from './error.types.js';
-import { authorize } from './authorize.request.js';
-import { AuthorizeErrorResponse, AuthorizeSuccessResponse } from './authorize.request.types.js';
-import { createAuthorizeOptions } from './authorize.request.utils.js';
+import type { AuthorizeErrorResponse } from './authorize.request.types.js';
+import type { TokenExchangeOptions } from './client.store.types.js';
+import type { TokenExchangeErrorResponse } from './exchange.types.js';
 
 export async function oidc<ActionType extends ActionTypes = ActionTypes>({
   config,
   requestMiddleware,
   logger,
+  storage,
 }: {
   config: OidcConfig;
   requestMiddleware?: RequestMiddleware<ActionType>[];
   logger?: {
     level: LogLevel;
     custom?: CustomLogger;
   };
+  storage?: Partial<StorageConfig>;
 }) {
   const log = loggerFn({ level: logger?.level || 'error', custom: logger?.custom });
+  const storageClient = createStorage({
+    storeType: 'localStorage',
```

Comment:
Should be `type` instead of `storeType`

Reply:
Not sure if you are agreeing or disagreeing with me here :) I think we are in agreement that `storeType` got changed to `type` in the `StorageConfig` interface. I'm saying line 44 should be changed to `type: 'localStorage'` to align with the new interface.

Reply:
Lol, sorry. I misread your comment. I will definitely change this.